ui/src/app/components/apps-panel.tsx

```diff
@@ -0,0 +1,406 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import {
+  faGear,
+  faPause,
+  faPlay,
+  faPlusCircle,
+  faTrashCan,
+} from '@fortawesome/free-solid-svg-icons';
+import { faSyncAlt } from '@fortawesome/free-solid-svg-icons/faSyncAlt';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { BillingStatus, RuntimeStatus, Workspace } from 'generated/fetch';
+
+import { Clickable } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import { DisabledPanel } from 'app/components/runtime-configuration-panel/disabled-panel';
+import { RuntimeStatusIcon } from 'app/components/runtime-status-icon';
+import { NewNotebookModal } from 'app/pages/analysis/new-notebook-modal';
+import colors from 'app/styles/colors';
+import { cond, reactStyles, switchCase } from 'app/utils';
+import { machineRunningCost, machineStorageCost } from 'app/utils/machines';
+import { formatUsd } from 'app/utils/numbers';
+import {
+  isActionable,
+  isVisible,
+  RuntimeStatusRequest,
+  toAnalysisConfig,
+  useRuntimeStatus,
+} from 'app/utils/runtime-utils';
+import {
+  diskStore,
+  runtimeStore,
+  serverConfigStore,
+  useStore,
+} from 'app/utils/stores';
+import jupyterLogo from 'assets/images/jupyter.png';
+import rStudioLogo from 'assets/images/RStudio.png';
+
+const styles = reactStyles({
+  expandedAppContainer: {
+    background: colors.white,
+    marginLeft: '1em',
+    marginBottom: '1em',
+    padding: '1em',
+    width: 'fit-content',
+  },
+  button: {
+    opacity: 1,
+    boxSizing: 'border-box',
+    height: 69,
+    width: 108,
+    border: '1px solid #979797',
+    borderRadius: 9,
+    alignItems: 'center',
+  },
+  disabledButton: {
+    opacity: 0.46,
+    cursor: 'not-allowed',
+    boxSizing: 'border-box',
+    height: 69,
+    width: 108,
+    border: '1px solid #979797',
+    borderRadius: 9,
+    alignItems: 'center',
+  },
+  enabledTrashButton: {
+    alignSelf: 'center',
+    marginLeft: 'auto',
+    marginRight: '1em',
+  },
+  disabledTrashButton: {
+    alignSelf: 'center',
+    marginLeft: 'auto',
+    marginRight: '1em',
+    color: colors.disabled,
+    cursor: 'not-allowed',
+  },
+  buttonText: {
+    color: '#4D72AA', // TODO fix
```

Comment:
Right - I got this from the mock, but it's weird that we're deviating from our normal palette here.  We typically use `colors.primary` or `colors.secondary` rather than spelling it out like this.  Maybe we can chat about this in the UX channel.

Here the text is in [#4D72AA, primary, secondary], for comparison.

<img width="413" alt="Screen Shot 2022-11-17 at 1 44 46 PM" src="https://user-images.githubusercontent.com/2701406/202531890-7207d033-ef68-499b-ba25-93ba8f5ca9e3.png">

Reply:
okay I think I like this combo: primary for enabled, secondary for disabled
<img width="409" alt="Screen Shot 2022-11-17 at 1 55 59 PM" src="https://user-images.githubusercontent.com/2701406/202534035-5ad4dc42-b00f-4361-ae56-cf93a3041b84.png">

Reply:
Interesting. I like the version you just posted.

Reply:
did you try colorWithWhiteness?

Reply:
I did not.  Not sure how much whiteness would be good here.